src/server-properties.cob

```diff
@@ -102,6 +107,15 @@ PROCEDURE DIVISION USING LK-FAILURE.
 
                     WHEN "motd"
                         MOVE FUNCTION TRIM(ENTRY-VALUE) TO SP-MOTD
+                    WHEN "max-players"
+                        MOVE FUNCTION NUMVAL(ENTRY-VALUE)
+                                   TO MAX-PLAYERS
+                                      MAX-CLIENTS
+                        IF MAX-PLAYERS IS NOT NUMERIC OR 
+                           MAX-PLAYERS < 1
+                           MOVE 1 TO LK-FAILURE
+                           GOBACK
+                        END-IF
```

Comment:
Please use fewer linebreaks. Also, it must be checked whether the string value (ENTRY-VALUE) is numeric, not the value already converted to a number (MAX-PLAYERS):

```suggestion
                        MOVE FUNCTION NUMVAL(ENTRY-VALUE) TO MAX-PLAYERS MAX-CLIENTS
                        IF ENTRY-VALUE IS NOT NUMERIC OR MAX-PLAYERS < 1
                           MOVE 1 TO LK-FAILURE
                           GOBACK
                        END-IF
```